mne/gui/_kit2fiff_gui.py

```diff
@@ -46,8 +47,8 @@
     hsp_wildcard = ['Head Shape Points (*.hsp;*.txt)|*.hsp;*.txt']
     elp_wildcard = ['Head Shape Fiducials (*.elp;*.txt)|*.elp;*.txt']
     kit_con_wildcard = ['Continuous KIT Files (*.sqd;*.con)|*.sqd;*.con']
-elif os.name == 'nt':
-    # on Windows, multiple wildcards does not seem to work
+elif sys.platform in ('win32',  'linux2'):
```

Comment:
It also has an effect on the UI with a difference in convenience – on macOS you can select one multi-wildcard setting and you see *all* matching files. On Windows and Ubuntu you can only see one file type at a time, and you have to select different wildcards in the file browser to see them.